ci/build-helpers/buildSrc/src/main/kotlin/org/jetbrains/compose/internal/publishing/utils/ModuleValidator.kt

```diff
@@ -1,115 +0,0 @@
-/*
```

Comment:
Validations is removed because:
- a sign validation is done on Maven Central side
- a jar validation is outdated (because we also publish klibs)